package.json

```diff
@@ -22,6 +22,7 @@
     "eslint": "^8.18.0",
     "node-fetch": "^3.2.6",
     "tsup": "^6.1.2",
-    "typescript": "^4.7.4"
+    "typescript": "^4.7.4",
+    "vsce": "^2.11.0"
```

Comment:
This was required in order to avoid needing vsce globally installed for the scripts